config/environments/development.rb

```diff
@@ -51,4 +51,7 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+  
+  # TODO: set :host in environments/production.rb to be the domain of our production app
+  config.action_mailer.default_url_options = { :host => 'justice.dev' }
```

Comment:
The host refers to the mail server. I would leave it to `config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }` in development.

Btw, this tool is great for testing emails locally: https://mailcatcher.me/